Applications/opensense/opensense.cpp

```diff
@@ -276,13 +276,16 @@ void PrintUsage(const char *aProgName, ostream &aOStream)
     aOStream << "-PropertyInfo, -PI                      Print help information for properties in setup files.\n";
     aOStream << "-ReadX, -RX  directory settings.xml     Parse Xsens exported files from directory using settingsFile.xml.\n";
     aOStream << "-ReadA, -RA  datafile.csv settings.xml  Parse single csv file provided by APDM using specified settingsFile.xml.\n";
-    aOStream << "-Calibrate, -C modelPoseFile.osim calibrationOrientations.sto.\n";
+    aOStream << "-Calibrate, -C modelPoseFile.osim calibrationOrientations.sto. <base_imu> <base_heading_axis>\n";
```

Comment:
This is minor, but in the setup file for IK, it is called base_imu_label ... should make consistent.